Tasks/AzurePowerShellV4/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 4,
         "Minor": 0,
-        "Patch": 5
+        "Patch": 6
```

Comment:
I am assuming we will update the patch version in master to 7 before M152 cut off.

Reply:
yes, I will do